src/mbgl/style/source_impl.cpp

```diff
@@ -22,8 +22,12 @@
 namespace mbgl {
 namespace style {
 
+namespace {
```

Comment:
Not necessary here; `static` data and functions are already non-visible beyond their translation unit.